tests_common/copy_files_from_core.py

```diff
@@ -52,8 +52,8 @@ def cleanup_dir(directory: str):
 simple_project_dir = os.path.join(data_dir, "simple_project")
 db_common_py = os.path.join(db_dir, "common.py")
 dest_db_common_py = os.path.join(dest_db_dir, "common.py")
-dump_tool_py = os.path.join(db_dir, "dump_tool.py")
-dest_dump_tool_py = os.path.join(dest_db_dir, "dump_tool.py")
+dump_tool_py = os.path.join(db_dir, "migration_tests", "dump_tool.py")
+dest_dump_tool_py = os.path.join(dest_db_dir, "migration_tests", "dump_tool.py")
```

Comment:
I don't see a reason why we want to package the `dump_tool.py` in `pytest-inmanta-extensions`. I think we can just remove it. @sanderr Do you see a reason why we need this?

Reply:
I think the same is true for the `common.py` actually.

Reply:
Ah no. This is not true for the `common.py`. It's used by inmanta-lsm. Maybe we just have to keep this in as it doesn't hurt to keep it in.

Reply:
I'm not aware of it being used anywhere.

Reply:
Ok. Let's kick it out then.